CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.3.1 (Apr 4, 2016)
+### general
+ - Fix a JRuby thread safety issue when using regular expression under multiple workers ([#4977](https://github.com/elastic/logstash/issues/4977)).
```

Comment:
in prior logs we use the first vern in the past tense "fixed", "disabled", ... we should be consistent